openslides/users/static/templates/users/group-list.html

```diff
@@ -15,48 +18,56 @@ <h1 translate>Groups</h1>
 </div>
 
 <div class="details">
-  <div class="row form-group">
-    <div class="col-sm-4 pull-right">
-      <div class="form-group">
-        <div class="input-group">
-          <div class="input-group-addon"><i class="fa fa-search"></i></div>
-          <input type="text" ng-model="filter.search" class="form-control"
-             placeholder="{{ 'Search' | translate}}">
-        </div>
-      </div>
-    </div>
-  </div>
-
-  <table class="table table-striped table-bordered table-hover">
+  <p class="lead" translate>
+    All your changes are saved immediatly. If the changes affect your permissions, you have to reload the page to observe the difference.
```

Comment:
Typo in first sentence: "immediately"
Suggestion for 2nd sentence to make it clear that it effects also for other users: "Changes you make are only effective once you (or the users concerned) reload the page."